### DISCUSSION TITLE
Question about android .apk version

### DISCUSSION THREAD

**Discussion by ubreathtaking** [pear-devs/pear-desktop#403] [Q&A]
Morning friends, is there a valid version of our beloved (th-ch/youtube-music) for the Android phone? Can't find.

**Comment by Araxeus** [comment]
you can use https://vancedapp.com/
(not related to this program, but great app nonetheless)

**Reply by ubreathtaking** [reply]
> you can use https://vancedapp.com/
> (not related to this program, but great app nonetheless)

Love you man, works great!
